dev/ComboBox/ComboBox_themeresources.xaml

```diff
@@ -382,6 +383,7 @@
         <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
         <Setter Property="FontSize" Value="{ThemeResource ControlContentThemeFontSize}" />
         <contract6Present:Setter Property="UseSystemFocusVisuals" Value="{ThemeResource IsApplicationFocusVisualKindReveal}" />
+        <contract7Present:Setter Property="primitives:ComboBoxAttachedProperties.ApplyDynamicCornerRadius" Value="true"/>
```

Comment:
>contract7Present:Setter [](start = 9, length = 23)

Why does this need contract7 to be present? Are we only doing rounding on RS5+?

Reply:
No it should work on lower versions, I removed the prefix and added interface check in code. They will get rounded corners on lower version but won't be to customize the value through CornerRadius property on ComboBox since it doesn't exist.